src/components/shared/Promotions/Promotions.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import { injectIntl } from 'react-intl';
+import { InjectAppServices } from '../../../services/pure-di';
+import Loading from '../../Loading/Loading';
+
+/**
+ * Promotions
+ * @param { Object } props
+ * @param { import('react-intl').InjectedIntl } props.intl
+ * @param { import('../../services/pure-di').AppServices } props.dependencies
+ */
+const Promotions = function({ intl, type, page, dependencies: { dopplerSitesClient } }) {
+  const [bannerData, setBannerData] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const defaultBannerData = {
+    title: _('default_banner_data.title'),
+    description: _('default_banner_data.description'),
+    backgroundUrl: _('default_banner_data.background_url'),
+    imageUrl: _('default_banner_data.image_url'),
+    functionality: _('default_banner_data.functionality'),
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      const bannerData = await dopplerSitesClient.getBannerData(intl.locale, type, page || '');
+      setBannerData(bannerData.success ? bannerData.value : defaultBannerData);
+      setIsLoading(false);
+    };
+
+    fetchData();
+  }, []);
```

Comment:
Do you think that _sites team_ will have the feature ready soon? if not, maybe you could commit something like this in order to allow us to merge your changes:

```js
const enableSitesContent = false;

useEffect(() => {
  if (enableSitesContent) {
    const fetchData = async () => {
      setIsLoading(true);
      const bannerData = await dopplerSitesClient.getBannerData(intl.locale, type, page || '');
      setBannerData(bannerData.success ? bannerData.value : defaultBannerData);
      setIsLoading(false);
    };
    fetchData();
  } else {
      setBannerData(defaultBannerData);
  } 
}, []);

```

Reply:
Another option is creating another implementation of the client, but, in this case, I think that this is easier.